service/src/main/java/uk/gov/hmcts/reform/fpl/service/docmosis/GeneratedOrderTemplateDataGeneration.java

```diff
@@ -41,6 +41,8 @@ public GeneratedOrderTemplateDataGeneration(
         this.localAuthorityNameLookupConfiguration = localAuthorityNameLookupConfiguration;
     }
 
+    // If we use <?, ?> there is then an issue with sonar complaining about not using wildcard types as return types
+    @SuppressWarnings("rawtypes")
```

Comment:
Any suggestions on this would be welcome

Reply:
for now, I think we can use the suppression until we can rethink the Docmosis object-ing approach

Reply:
Yeah, the issue is with the use of `@SuperBuilder`. My suggestion would be to move to how we have started doing it with the notifications. There we don't use the builder pattern but instead just use setter methods. If we wanted to, we could even instead of doing lombok setters we could make custom setters that return the object as well allowing chaining of the setter methods.

Reply:
Actually, looks like lombok supports this as well
>lombok.accessors.chain = [true | false] (default: false)
If set to true, generated setters will return this (instead of void). An explicitly configured chain parameter of an @Accessors annotation takes precedence over this setting.

Reply:
yes, something we can definitely explore. Do you have a Tech Enhancement ticket for it?

Reply:
not yet